must-gather/collection-scripts/gather_cluster_logging_operator_resources

```diff
@@ -39,9 +38,5 @@ oc -n $NAMESPACE get deployments -o wide > ${clo_folder}/deployments.txt 2>&1
 oc -n $NAMESPACE get ds -o wide > ${clo_folder}/daemonsets.txt 2>&1
 oc -n $NAMESPACE get pods -o wide > ${clo_folder}/pods.txt 2>&1
 oc -n $NAMESPACE extract secret/elasticsearch --to=${clo_folder}
-oc -n $NAMESPACE extract configmap/fluentd --to=${clo_folder}
 oc -n $NAMESPACE extract configmap/collector --to=${clo_folder}
-oc -n $NAMESPACE extract configmap/secure-forward --to=${clo_folder}
-oc -n $NAMESPACE extract secret/secure-forward --to=${clo_folder}
-oc -n $NAMESPACE get events --field-selector involvedObject.kind=ClusterLogging -o yaml > ${clo_folder}/clo-events.yaml 2>&1
-oc -n $NAMESPACE get events --field-selector involvedObject.kind=ClusterLogForwarder -o yaml > ${clo_folder}/clf-events.yaml 2>&1
+oc -n $NAMESPACE get events -o yaml > ${clo_folder}/$NAMESPACE-events.yaml 2>&1
```

Comment:
do you want to add `-o yaml` here?

Reply:
it was in original command, so decide to keep it, in the same time i think in not necessary